ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -543,6 +543,8 @@ keys:
             - network
         vmtracer:
           type: bool
+        ptp_mpass:
```

Comment:
mpass should be added under the ptp key below.

Reply:
Updated eos_designs part as well